integration/mediation-tests/tests-other/src/test/resources/artifacts/ESB/hotdeployment/deployment.toml

```diff
@@ -1,6 +1,6 @@
 [server]
 hostname = "localhost"
-hot_deployment = "true"
+hot_deployment = "false"
```

Comment:
Shall we use the boolean value instead of a String here as well?

Reply:
In other places this is given as boolean, I left this this way just to make sure that both string and boolean value works for hot deployment. wdyt.